corehq/apps/registration/forms.py

```diff
@@ -521,6 +534,8 @@ def __init__(self, data=None, excluded_emails=None, is_add_user=None, location=N
                     data_bind="textInput: email",
                 ),
                 'role',
+                'profile' if 'profile' in self.fields else None,
```

Comment:
How were these being displayed before? And why doesn't `program` need to be in here?

Reply:
I found that strange too, i don't know how `supply_point` / `location` and `program` are ever shown. But I do see invites created in the past 3 months with non-null values for those fields. Possibly those were all from bulk upload? 

The layout was added in [this commit ](https://github.com/dimagi/commcare-hq/commit/e1c619f2d67ce2bedc0b2adae720f6e9023acd0c)3 years ago so, I decided to just leave it as is for those fields and not mess with it within the scope of this PR.

Reply:
Makes sense to me.